applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_analysis.py

```diff
@@ -175,6 +180,11 @@ def _ExecuteInitialize(self):
         for process in self.list_of_processes:
             process.ExecuteInitialize()
 
+        ## Add the processes to the solver
+        self.solver.AddProcessesList(self.list_of_processes)
```

Comment:
This is a serious blocker
Not too long ago and after very long discussions with @KratosMultiphysics/technical-committee and others it was decided that the processes live in the stage, not the solver

Reply:
True, then I will need to add it to the process info in a similar way as is done in #2060